configure

```diff
@@ -829,8 +829,12 @@ SRCDIRS
 THREADHEADERS
 PANEL_LIBS
 PANEL_CFLAGS
-CURSES_LIBS
-CURSES_CFLAGS
+NCURSES_LIBS
+NCURSES_CFLAGS
+PANELW_LIBS
+PANELW_CFLAGS
+NCURSESW_LIBS
+NCURSESW_CFLAGS
```

Comment:
This is unfortunate, but it is autogenerated by the `PKG_CHECK_MODULES` macro.

Reply:
Quoting from the `pkg-config` man page (emphasis mine):

> PKG_CHECK_MODULES(VARIABLE-PREFIX, MODULES [,ACTION-IF-FOUND [,ACTION-IF-NOT-FOUND]])
> The macro PKG_CHECK_MODULES can be used in configure.ac to check whether modules exist. [...]
>
> This would result in MYSTUFF_LIBS and MYSTUFF_CFLAGS substitution variables, set to the libs and cflags for the given module list.  If a module is missing or has the wrong version, by default configure will abort with a message. To replace the default action, specify an ACTION-IF-NOT-FOUND. PKG_CHECK_MODULES will not print any error messages if you specify your own ACTION-IF-NOT-FOUND.  However, it will set the variable MYSTUFF_PKG_ERRORS, which you can use to display what went wrong.
>
> [...]
>
> **Also note that repeated usage of VARIABLE-PREFIX is not recommended.  After the first successful usage, subsequent calls with the same VARIABLE-PREFIX will simply use the _LIBS and _CFLAGS variables set from the previous usage without calling pkg-config again.**

We might work around this by explicitly unsetting `CURSES_LIBS` and `CURSES_CFLAGS` between calls. I _guess_ it would work, but it sure feels hacky. OTOH, it would make for a better `configure` user experience.

Reply:
Getting back to this, I see now that we can safely call `PKG_CHECK_MODULES` with the same variable-prefix; the second call is only done if the first fails, and if the first fails, no variables are set.